### PR TITLE
Split Up Writable Interface

### PR DESCRIPTION
**Pull request by adrian-enspired**
**BASED ON** https://github.com/nexcess/nexcess-php-sdk/pull/9

fixes https://nexcess.atlassian.net/browse/NSD-11733

### CODE REVIEW

**Comment by adrian-enspired** [comment]
fixed conflict

File: src/Resource/Create.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use Nexcess\Sdk\ {
+  ApiException,
+  Resource\Model
+};
+
+/**
+ * Provides create() functionality for API endpoints.
+ * Endpoints using this trait MUST implement Creatable.
+ */
+trait Create {
```
Comment (robertology) [inline src/Resource/Create.php]: suggest `canCreate`, `canDelete`, etc

Comment (adrian-enspired) [reply src/Resource/Create.php]: done

**Comment by calevans** [comment]
Conceptionally, this looks good.

**Pull request merged**